### DISCUSSION TITLE
Budibase resets upon machine restart

### DISCUSSION THREAD

**Discussion by rajat-b** [Budibase/budibase#8606] [Help]
Hi everybody.

I had made an app using budibase installed on my windows machine (running `wampserver`).  However, when I restarted the machine, the budibase builder is redirecting to `http://localhost:10000/builder/admin`. I had already created admin, of course. I fear if my ~10 hours spent work have been lost.

Is there any way to recover it? If not, will this repeat everytime the machine restarts? Obviously it shouldn't - I am surely missing something.

**Comment by melohagan** [comment]
Hey @rajat-b 

It seems that the docker volume that holds all of the Budibase data has perhaps got wiped. You could try doing a `docker volume ls` to see if it's there.

Otherwise, I recommend in future making use of the backup functionality: https://docs.budibase.com/docs/backups
